app/src/androidTest/java/org/codedoesgood/androidtemplate/UtilityTests/Tests/DateTimeFormatTest.java

```diff
@@ -0,0 +1,78 @@
+package org.codedoesgood.androidtemplate.UtilityTests.Tests;
+
+import android.content.Context;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.codedoesgood.androidtemplate.MainActivity;
+import org.codedoesgood.androidtemplate.R;
+import org.codedoesgood.androidtemplate.UtilityTests.Robots.DateTimeFormatRobot;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Calendar;
+
+import static junit.framework.Assert.assertTrue;
+
+@RunWith(AndroidJUnit4.class)
+public class DateTimeFormatTest {
```

Comment:
Good Test class but I believe it's better to use Roboeletric for Unit Test accessing the Android Framework, with Roboeletric the Test becomes faster. Site: http://robolectric.org/